.github/workflows/sast-and-package-prod.yml

```diff
@@ -30,7 +32,7 @@ jobs:
         languages: Java    
         
     - name: Build with Maven
-      run: mvn clean install -Denvironment=prod
+      run: mvn clean install -DENV_VAR=${{ env.ENV_VAR }}
```

Comment:
We are not supposed to change other files unless there is a bug or strong use case. Revert this change

Reply:
This update was done by @Vidyaaa24 on this commit: https://github.com/beehyv/PSMRI-HWC-API/commit/8b71c28c9d1a873b76bae3311fcf25ecfea097b6